src/releaseNotesBuilder.ts

```diff
@@ -44,7 +45,10 @@ export class ReleaseNotesBuilder {
     // load octokit instance
     const octokit = new Octokit({
       auth: `token ${this.token || process.env.GITHUB_TOKEN}`,
-      baseUrl: `${this.baseUrl || 'https://api.github.com'}`
+      baseUrl: `${this.baseUrl || 'https://api.github.com'}`,
+      request: {
+        agent: new ProxyAgent()
+      }
```

Comment:
It would be great if we only set-up the `ProxyAgent()` in case it is configured to be used - can have a look into this.